combat/combatSystem.cpp

```diff
@@ -426,6 +572,53 @@ int Player::getDefenseSkill() const {
         return(defenseSkill->getGained() + bonus);
 }
 
+double Creature::getMisschanceModifier(const std::shared_ptr<Creature>& victim, double& missChance) {
```

Comment:
```suggestion
double Creature::getMissChanceModifier(const std::shared_ptr<Creature>& victim, double& missChance) {
```
References would need to be updated as well

Reply:
new function so never been called til now